### PR TITLE
adding more info

### PR DESCRIPTION
**Pull request by get-itips**
closes https://github.com/MicrosoftDocs/OfficeDocs-SkypeForBusiness/issues/8197

@HowlinWolf-92 Can you please review? Thank you

### CODE REVIEW

**Comment by scanum** [comment]
@JohanFreelancer9 Copy editing is needed for this PR. Thanks.

File: Teams/live-components-in-teams.md [hunk 1]
```diff
@@ -93,7 +93,7 @@ PS C:\\WINDOWS\\system32&gt;
 
 - Live components are disabled in federated chats.
 
-- B2B guests will not be able to collaborate on live component that is shared with them via Company Share Link unless tenant sets external access option to allow B2B guests have same access level as tenant members.
+- B2B guests will not be able to collaborate on live component that is shared with them via Company Share Link unless tenant sets external access option to allow B2B guests have same access level as tenant members. More information, see [Configure B2B external collaboration settings](/azure/active-directory/external-identities/delegate-invitations#configure-b2b-external-collaboration-settings).
```
Comment (JohanFreelancer9) [inline Teams/live-components-in-teams.md:96]:
```suggestion
- B2B guests will not be able to collaborate on a live component that is shared with them via Company Share Link unless the tenant sets an external access option to allow B2B guests to have the same access level as tenant members. For more information, see [Configure B2B external collaboration settings](/azure/active-directory/external-identities/delegate-invitations#configure-b2b-external-collaboration-settings).
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by scanum** [comment]
@HowlinWolf-92 PR has been copyedited and is ready for final review, could you please check and merge? Thanks!

**Comment by get-itips** [comment]
@tiburd Please help me to get this merged.

**Comment by tiburd** [comment]
Reviewed + confirmed changes.

**Pull request merged**